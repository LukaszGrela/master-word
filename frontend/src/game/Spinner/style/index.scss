@import '../../../styles/variables';

@import './Spinner';
