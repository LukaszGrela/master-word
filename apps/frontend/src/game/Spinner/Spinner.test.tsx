import { describe, expect, it } from 'vitest';
import Spinner from './Spinner';
import { render } from '@testing-library/react';

describe('game', () => {
  describe('Spinner', () => {
    it('Renders', () => {
      const { container } = render(<Spinner />);

      expect(container).toMatchSnapshot();
    });
    it('Renders with all props', () => {
      const { container } = render(<Spinner className="test" />);

      expect(container).toMatchSnapshot();
    });
  });
});
