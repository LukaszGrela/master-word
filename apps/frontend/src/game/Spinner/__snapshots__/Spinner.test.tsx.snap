// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`game > Spinner > Renders 1`] = `
<div>
  <div
    class="Spinner"
  >
    <svg
      class="Spinner_svg"
      height="40"
      version="1"
      viewBox="0 0 40 40"
      width="40"
    >
      <circle
        cx="20"
        cy="20"
        id="bg"
        r="18"
      />
      <circle
        cx="20"
        cy="20"
        id="fg"
        r="18"
      />
    </svg>
  </div>
</div>
`;

exports[`game > Spinner > Renders with all props 1`] = `
<div>
  <div
    class="Spinner test"
  >
    <svg
      class="Spinner_svg"
      height="40"
      version="1"
      viewBox="0 0 40 40"
      width="40"
    >
      <circle
        cx="20"
        cy="20"
        id="bg"
        r="18"
      />
      <circle
        cx="20"
        cy="20"
        id="fg"
        r="18"
      />
    </svg>
  </div>
</div>
`;
